js/theme.js

```diff
@@ -238,3 +238,33 @@ if (typeof(window) != 'undefined') {
             clearTimeout(id);
         };
 }());
+
+// function for opening/closing sidebar on
+// swiping right/close.
+(function () {
+    var start = null;
+      window.addEventListener("touchstart",function(event){
```

Comment:
These functions should go into `ThemeNav` and the listeners added during the `init()` call